tests/rt.sh

```diff
@@ -707,16 +726,20 @@ elif [[ $MACHINE_ID = orion ]]; then
   module load gcc/10.2.0
   module load python/3.9.2
 
-  module load contrib rocoto
-  ROCOTORUN=$(which rocotorun)
-  ROCOTOSTAT=$(which rocotostat)
-  ROCOTOCOMPLETE=$(which rocotocomplete)
-  ROCOTO_SCHEDULER=slurm
+  if [[ "${ROCOTO:-false}" == true ]] ; then
+    module load contrib rocoto
+    ROCOTORUN=$(which rocotorun)
+    ROCOTOSTAT=$(which rocotostat)
+    ROCOTOCOMPLETE=$(which rocotocomplete)
+    ROCOTO_SCHEDULER=slurm
+  fi
 
   module use /work/noaa/epic/role-epic/spack-stack/orion/modulefiles
```

Comment:
I'm not sure, but I believe this 'module use' is only needed for loading ecflow module from this specific location. Needs to be tested and moved inside ecflow if block. Later.